### PR TITLE
Rename E Function Params - entityLoad()

### PR DESCRIPTION
**Pull request by shawnoden**
This one has a complex signature with 2 sets of optional params that are mutually exclusive.

### CODE REVIEW

File: data/en/entityload.json [hunk 1]
```diff
@@ -1,17 +1,17 @@
 {
 	"name":"entityLoad",
 	"type":"function",
-	"syntax":"entityLoad(entityName [, filterCriteria] [, unique])\n entityLoad(entityName [, filterCriteria] [, sortOrder] [, options])",
+	"syntax":"EntityLoad(entityName [,id | Filter ,unique | Order ,options])",
```
Comment (pfreitag) [inline data/en/entityload.json]: You are changing the case of `entityLoad` to `EntityLoad`, the docs all use `lowerCamelCase` for function names.

Comment (shawnoden) [reply data/en/entityload.json]: I prefer lowerCamelCase too, but NamedParametersColdFusion2018.pdf showed it to have a leading capital. I know we were changing case in the param names, but I think that initial cap may have been an unintentional outcome of the document they generated. If the casing of the function name doesn't matter, and our other function names are primarily lowerCamelCase, then I'll revert that change.

Comment (shawnoden) [reply data/en/entityload.json]: Done.

**Comment by pfreitag** [comment]
Thanks

**Pull request merged**